front/pages/api/v1/w/[wId]/spaces/[spaceId]/data_sources/[dsId]/documents/[documentId]/parents.ts

```diff
@@ -53,6 +53,9 @@ import { apiError } from "@app/logger/withlogging";
  *           schema:
  *             type: object
  *             properties:
+ *               parentId:
+ *                 type: string
+ *                 description: Direct parent document IDs
```

Comment:
It's not really a document ID as it can be a folder or a table. We should update the comment for parents as well.